_posts/wookiemage/2016-04-28-Colins-Final-Project.md

```diff
@@ -0,0 +1,54 @@
+
+[Regex Crossword](https://regexcrossword.com/) Here’s one that I don’t understand at all. The goal is to take the obscurity of regexs and turn it into a game. There are lots of levels that use different features of regexs. It really tests your knowledge.
+
+Some Community Resources:
+
+(Learn Python Reddit)[https://www.reddit.com/r/learnpython/] This subreddit has over 50k subscribers of differing ability in Python who all have the same goal, to help others learn python. While I haven’t posted any questions to this subreddit, I have read a lot of great questions and explanations there. As with any subreddit, you get the bad with the good, so sometimes you see very terse answers with little explanation, but generally those get downvoted and nice verbose answers get upvoted. 
+
+Another great feature of this community is our next resource:
+
+(The Learn Python Subreddit Wiki)[https://www.reddit.com/r/learnpython/wiki/index] This is a long list of community resources generated and maintained by the learnpython group. It has lots of different categories of resources, everything from video tutorials, to explanations of the differences between python 2 and 3. This is a good place to turn when google fails you, or you want some curated resources.
+
+## A Semester in review
+
+I’ve really learned a lot in this course. While I had some programming knowledge before we started, which was very helpful, I’ve really built up my skills this semester. Using turtle to work through flow of control and as a simple graphical user interface was a good scaffold. I am glad we had the chance to get our grounding in something a little less intimidating that the command line. I’m also glad we had the chance to progress on the command line and learn how to use a web based IDE. Similarly, building skills in github was very good practice for the future. I have a much better understanding of how a tool like github can help remote teams collaborate or even just manage version control for local teams.
+
+One thing that I would like would be a list of further resources and project ideas to continue to push ourselves further. I feel like a have a good grasp of the basics of python, but I’m not really sure where to go next. I know this is a common feeling among new programmers, as there are lots and lots of threads about this exact topic on the learn python subreddit. The answer always seems to be a variation of “keep going” or “make something for yourself.” I think a lot of people find this answer unsatisfying. Maybe we can make some sort of list of project ideas as a community effort. Or, more likely, such a thing already exists and I haven’t run into it yet. In fact, one small google search later, I found [this](https://www.reddit.com/r/dailyprogrammer). I haven’t had a chance to assess whether these challenges are appropriate, but they're a place to start.
```

Comment:
Great idea!  I'll add a final assignment, Keep Going with some resources that others can add to.  When I do, consider adding the links you shared here to it (it'll be in /_posts/exercises)